File: Array/027/027.py
class Solution(object):

    def removeElement(self, nums, val):
        """
        :type nums: List[int]
        :type val: int
        :rtype: int
        循环判断，一直删除到没有（因为remov方法每次只会删除第一个）
        """
        while val in nums:
            nums.remove(val)
        return len(nums)

File: Array/027/027.js
/**
 * @param {number[]} nums
 * @param {number} val
 * @return {number}
 * 使用类似于两个指针的思路
 */
function removeElement (nums, val) {
  let j = 0;
  for (let i=0; i<nums.length; i++) {
    if (nums[i] !== val) {
      nums[j++] = nums[i];
    }
  }
  return j;
}
